llvm/test/CodeGen/RISCV/rvv/vsetvli-insert.ll

```diff
@@ -630,8 +630,9 @@ define void @add_v16i64(ptr %x, ptr %y) vscale_range(2,2) {
 define <vscale x 2 x float> @fp_reduction_vfmv_s_f(float %0, <vscale x 8 x float> %1, i64  %2) {
 ; CHECK-LABEL: fp_reduction_vfmv_s_f:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    vsetvli zero, a0, e32, m4, ta, ma
+; CHECK-NEXT:    vsetvli zero, a0, e32, m1, ta, ma
 ; CHECK-NEXT:    vfmv.s.f v12, fa0
+; CHECK-NEXT:    vsetvli zero, a0, e32, m4, ta, ma
```

Comment:
This is a regression. Do you have a plan to fix this?

Reply:
I think @preames's [draft patch here](https://github.com/llvm/llvm-project/pull/75544#issuecomment-1858133214) should fix it. Happy to take over it if there's no objections

Reply:
https://github.com/llvm/llvm-project/pull/76801